src/dbg/_exports.cpp

```diff
@@ -157,24 +158,35 @@ static bool getLabel(duint addr, char* label)
         }
         if(!retval)  //search for module entry
         {
-            duint entry = ModEntryFromAddr(addr);
-            if(entry && entry == addr)
+            if(addr != 0 && ModEntryFromAddr(addr) == addr)
             {
-                strcpy_s(label, MAX_LABEL_SIZE, "EntryPoint");
-                retval = true;
+                strcpy_s(label, MAX_LABEL_SIZE, GuiTranslateText(QT_TRANSLATE_NOOP("DBG", "EntryPoint")));
+                return true;

```

Comment:
This is also incorrect. The if statement should fall through so other types can be queried at the same time and also allow plugins to use `CB_ADDRINFO`.